minio/commonconfig.py

```diff
@@ -417,18 +417,6 @@ def __init__(self, bucket_name, object_name, region=None, version_id=None,
 
     def _validate_size(self, object_size):
         """Validate object size with offset and length."""
-        def _get_error(name, value):
```

Comment:
Use this function below in all `raise ValueError()`

Reply:
could you please provide an example of how it should be used?

Reply:
`raise ValueError(_get_error("offset", self._offset))`

Reply:
changed

Reply:
Please do required changes than changing function name and it's behaviour.

Reply:
I did change the function, because I see an unnecessary repetition of ValueError. Also, I don't see a need to add underscore to the function name, because it's internal and is not visible anywhere outside of this particular method scope